.github/workflows/pr-check_markdownlint.yml

```diff
@@ -34,5 +43,4 @@ jobs:
       - name: Lint markdown files
         run: |
           echo "::add-matcher::.github/workflows/markdownlint-problem-matcher.json"
-          files_to_lint="${{ steps.changed-files.outputs.all_changed_files }}"
-          yarn markdownlint-cli2 ${files_to_lint}
+          yarn markdownlint-cli2 ${{ env.DIFF_DOCUMENTS }}
```

Comment:
It'll throw syntax error when there are files with `(` in their path. Refer https://github.com/mdn/content/pull/20466
The indirection was used to avoid the syntax error.

For example,
```
# this doesn't work
touch file(css).md

# this does
touch "file(css).md"
```

Reply:
```suggestion
          files_to_lint="${{ env.DIFF_DOCUMENTS }}"
          yarn markdownlint-cli2 ${files_to_lint}
```

Reply:
Thanks. Fixed in 9395ebad4a180e94913f6631326aa016978f4432